src/qt/sendcoinsentry.h

```diff
@@ -25,7 +25,7 @@ namespace Ui {
  * Stacked widget, with different UIs for payment requests
```

Comment:
Please change the comment too

Reply:
@laanwj 

Thank you! [Updated](https://github.com/bitcoin-core/gui/pull/612#issuecomment-1153138515).